website/integrations/services/owncloud/index.md

```diff
@@ -0,0 +1,210 @@
+            - **Redirect URI**:
+                - Strict: `oc://android.owncloud.com`
+
+            **iOS Application**
+
+            - **Signing Key**: Select any available signing key.
+            - **Client ID**: Use the predefined value found in the [ownCloud admin manual](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html#client-id).
+            - **Client Secret**: Use the predefined value found in the [ownCloud admin manual](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html#client-secret).
+            - **Redirect URI**:
+                - Strict: `oc://ios.owncloud.com`
+
+        - **Advanced protocol settings:**
+            - **Scopes**: Select the following scopes for each of the four application/provider pairs: `email`, `offline_access`, `openid`, `profile`.
+
+    - **Configure Bindings** (optional): To manage the listing and access to applications on a user's **My applications** page, you can optionally create a [binding](https://docs.goauthentik.io/docs/add-secure-apps/flows-stages/bindings/) between the application and a specific policy, group, or user. Keep in mind that if you do not specify any bindings, all users will have access to the application. For more information about user access, refer to our documentation about [authorization](https://docs.goauthentik.io/docs/add-secure-apps/applications/manage_apps#policy-driven-authorization) and [hiding an application](https://docs.goauthentik.io/docs/add-secure-apps/applications/manage_apps#hide-applications).
+
+### Service discovery
+
+To allow ownCloud applications to log in via OIDC, your reverse proxy must be configured to rewrite `https://owncloud.company/.well-known/openid-configuration` to `https://owncloud.company/index.php/apps/openidconnect/config`.
+
+Refer to the [ownCloud Admin Manual](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html#set-up-service-discovery) for an example configuration using [Apache HTTPD](https://httpd.apache.org/).
+
+For other reverse proxies, consult the provider-specific documentation for guidance on implementing this rewrite rule.
+
+## ownCloud Configuration
+
+To enable OIDC functionality in ownCloud, follow these steps:
+
+1. **Navigate to the Market**:
+
+    - Access the Market by visiting:  
+      `https://owncloud.company/apps/market/#/`  
+      or by clicking the **Hamburger Menu** in the top-left corner of any page in your ownCloud deployment and selecting **Market**.
+    - Search for and enable the **OIDC plugin**.
+
+2. **OIDC Plugin Configuration**:
+   The OIDC plugin cannot be configured via the ownCloud UI. Configuration must be performed either:
+
+    - Through the `config.php` file, or
+    - By storing the configuration in the ownCloud database.
+
```

Comment:
That's a kinda unconventional way to show two choices, but the bullet points are good and make it easy to see the two.

I'd suggest this though really we don't need the "OR":

```
The OIDC plugin cannot be configured via the ownCloud UI. Configuration must be performed either:

    - by editing the `config.php` file
    OR
    - by storing the configuration in the ownCloud database
```